### DISCUSSION TITLE
Filament Spatie Media Library can't fetch images from google drive disk in edit form

### DISCUSSION THREAD

**Discussion by alkoumi** [filamentphp/filament#8639] [Help]
Filament Spatie Media Library can't fetch file images from Google Drive disk in edit form and getting  2 errors CORS &  net::ERR_FAILED 403 but working fine in uploading files and infolists 

![Screenshot 1445-03-06 at 11 03 05 AM](https://github.com/filamentphp/filament/assets/10585943/9e69f2d6-696c-4d30-a3f9-f9e8a03c05d3)

Any one can Help!

**Comment by krekas** [comment]
The error is very clear. The problem is with your setup for google drive. It's not related to filament
